packages/app/package.json

```diff
@@ -43,7 +43,7 @@
     "@backstage/plugin-techdocs-react": "^1.1.3",
     "@backstage/plugin-user-settings": "^0.7.0",
     "@backstage/theme": "^0.2.17",
-    "@parodos/plugin-parodos": "0.4.0",
```

Comment:
changesets would have updated this dependency.  If it is left to manual, we always forget.

Reply:
since we are in the same project, we do not need to use npm-downloaded version at all, just reference the package locally.
That would be better, imho. We would always use up-to-date code when the wrapping app is executed.

Reply:
If the version matches the version in the monorepo then it is a symlinked version as it is yarn workspaces.  If it us not, it goes to npm